src/screens/ImportWalletScreen.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import { NavigationScreenProps } from 'react-navigation';
+
+import { Header, TextAreaItem, FlatButton } from 'app/components';
+import { Button } from 'app/components/Button';
+import { Route } from 'app/consts';
+import { CreateMessage, MessageType } from 'app/helpers/MessageCreator';
+import { en } from 'app/locale';
+import { NavigationService } from 'app/services';
+import { typography, palette } from 'app/styles';
+
+import {
+  SegwitP2SHWallet,
+  LegacyWallet,
+  WatchOnlyWallet,
+  HDSegwitP2SHWallet,
+  HDLegacyP2PKHWallet,
+  HDSegwitBech32Wallet,
+} from '../../class';
+
+const BlueApp = require('../../BlueApp');
+const EV = require('../../events');
+const loc = require('../../loc');
```

Comment:
you can use imports from `app/legacy` now :)

Reply:
Superb :)